Library/Formula/weechat.rb

```diff
@@ -30,12 +34,13 @@ def install
 
     args = std_cmake_args
 
-    args << "-DENABLE_LUA=OFF"    if build.without? "lua"
-    args << "-DENABLE_PERL=OFF"   if build.without? "perl"
-    args << "-DENABLE_RUBY=OFF"   if build.without? "ruby"
-    args << "-DENABLE_ASPELL=OFF" if build.without? "aspell"
-    args << "-DENABLE_GUILE=OFF"  if build.without? "guile"
-    args << "-DENABLE_PYTHON=OFF" if build.without? "python"
+    args << "-DENABLE_LUA=OFF"        if build.without? "lua"
```

Comment:
Reduce the spacing on these args: `args << "-DENABLE_LUA=OFF" if build.without? "lua"` is fine.